### DISCUSSION TITLE
Documentation for msprime 0.7.3

### DISCUSSION THREAD

**Discussion by paruljohri** [tskit-dev/tskit#1699] [Q&A] [answered]
Hi!

I was wondering where I could still access the documentation for msprime version 0.7.3. 

Thank you so much!

**Comment by petrelharp** [comment, marked as answer]
They're not available online any more (that I can find), but you should be able to build them yourself:
```
git clone https://github.com/tskit-dev/msprime.git
cd msprime
pip install -r requirements/readthedocs.txt
cd docs
git checkout 0.7.3
make
```
... and then you can read them by directing your web browser to the `_build/html` subdirectory.

I've just tested this and it worked for me; any problems will probably come from the third step (ie getting the right version of the python packages).

**Reply by paruljohri** [reply]
Thank you so much!

**Reply by petrelharp** [reply]
If it works, perhaps mark this as the answer?
